addons/addons.json

```diff
@@ -182,5 +181,6 @@
   "editor-buttons-reverse-order",
   "columns",
   "editor-compact",
-  "editor-theme3"
+  "editor-theme3",
+  "find-references"
```

Comment:
New addons usually go above themes. Have you checked this order is necessary?

Reply:
I don't know that this order matters. I was wondering why the colorscheme changed not work on my addon. 
Seems this is the reason.